docs/tutorial/robots.rst

```diff
@@ -141,6 +141,36 @@ of joints:
 * **Fixed**: Not really a joint because it cannot move, all degrees of freedom
   are locked.
 
+
+Visualizing Robots
+==================
+
+Before jumping into how to build a robot model, let's first see how to visualize
+one. This can be done with Blender, Rhino or Grasshopper using one of COMPAS's
```

Comment:
Not sure how it works with acronyms, is it `COMPAS's` or `COMPAS'`?

Reply:
There are different schools of thought on this regardless of whether the possessing noun is an acronym.  Are we going to go with AP or Chicago? https://www.thepunctuationguide.com/style.html. To me, COMPAS's sounds righter, but probably because the plural of compass is compasses, which is totally irrelevant.

Reply:
According to another site:

> In 2019, the AP raised quite the ruckus when they tweeted that they were considering adding an S after the apostrophe for singular proper nouns, as in Mavis Staples’s album or Martha Reeves’s concert. To date, no changes have been made, but as you can see, it’s an ever-evolving, highly volatile topic. 

So we will be wrong/out of date no matter what.  I will merge.